lib/generators/rolemodel/testing/rspec/templates/rails_helper.rb

```diff
@@ -60,4 +60,13 @@
   config.filter_rails_from_backtrace!
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
+
+  config.before :example, type: :system do
```

Comment:
This something we need to do across the board? Might need to not which generators were changed as part of this effort in the readme so we know what we need to run for an upgrade

Reply:
Leaving for now